src/util/getActiveTool.js

```diff
@@ -0,0 +1,28 @@
+import { getters, state } from '../store';
+import getActiveToolsForElement from '../store/getActiveToolsForElement';
+import filterToolsUseableWithMultiPartTools from '../store/filterToolsUsableWithMultiPartTools';
+
+// Todo: We could simplify this if we only allow one active
+// Tool per mouse button mask?
+export default function getActiveTool(eventData) {
```

Comment:
1. Can we update the params here to be a bit more explicit instead of taking the entire event?
    - IE. `element` and `buttons`
2. Can we rename this to something more explicit? `getActiveMouseTool` or we can keep `getActiveTool`, but provide overloads where one of the params is `InteractionType` (mouse or touch)